tensorflow_datasets/audio/commonvoice_test.py

```diff
@@ -0,0 +1,18 @@
+"""
+Mozilla CommonVoice Dataset Builder UnitTest
+"""
+import tensorflow_datasets.testing as tfds_test
+from tensorflow_datasets.audio import commonvoice
+
+
+class CommonVoiceTest(tfds_test.DatasetBuilderTestCase):
+    """
+    CommonVoice Tester Class
+    """
+    DATASET_CLASS = commonvoice.CommonVoice
+    SPLITS = {"train": 10, "test": 10, "validation": 10}
```

Comment:
Let's try to have fewer test files. Could these be, for example, 3 each?

Reply:
Sure. That Can be done.

Reply:
Resolved.